components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuFeature.kt

```diff
@@ -42,14 +43,17 @@ internal const val FRAGMENT_TAG = "mozac_feature_contextmenu_dialog"
  * @property engineView The [EngineView]] this feature component should show context menus for.
  * @param tabId Optional id of a tab. Instead of showing context menus for the currently selected tab this feature will
  * show only context menus for this tab if an id is provided.
+ * @param showAdditionalNote Optional flag used to decide if an additional note should be attached to the menu or not
  */
+@Suppress("LongParameterList")
 class ContextMenuFeature(
     private val fragmentManager: FragmentManager,
     private val store: BrowserStore,
     private val candidates: List<ContextMenuCandidate>,
     private val engineView: EngineView,
     private val useCases: ContextMenuUseCases,
-    private val tabId: String? = null
+    private val tabId: String? = null,
+    private val showAdditionalNote: Boolean = false
```

Comment:
The note we add is very specific to Firefoc Focus (it even contains the app name), so I'm wondering instead of a `Boolean`, how about we allow the app to define the note. We could pass in a `String`, but the choice to only show it for specific items, is also very app-dependant. So maybe we can have something like:

```Kotlin
private val additionalNote: (HitResult) -> String?
```

Then it's up to the app to define the behavior. And the string can come from the app. :)